lib/isaac.go

```diff
@@ -130,6 +130,7 @@ func (is *ISAAC) receiveBallotStateINIT(ballot Ballot) (vs VotingStateStaging, e
 		is.Boxes.WaitingBox.RemoveVotingResult(vr) // TODO detect error
 		if !vs.IsClosed() {
 			is.Boxes.VotingBox.AddVotingResult(vr) // TODO detect error
+			is.Boxes.AddSource(ballot)
```

Comment:
Calling `AddSource` only here?
I figure it is also called in https://github.com/bosnet/sebak/pull/119/files#diff-2231e829007317368e86620b46b6bfcaR343. Did I miss something?

Reply:
@Charleslee522 I think your opinion is right.